stat-svc/service/src/main/java/ru/practicum/explorewithme/statsvc/service/config/JacksonConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package ru.practicum.explorewithme.statsvc.service.config;
+
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
+import org.springframework.boot.autoconfigure.jackson.Jackson2ObjectMapperBuilderCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.time.format.DateTimeFormatter;
+
+@Configuration
+public class JacksonConfiguration {
+    @Bean
+    public Jackson2ObjectMapperBuilderCustomizer jackson2ObjectMapperBuilderCustomizer() {
```

Comment:
Можно сделать короче, чтобы инжектить по более короткому имени, тем более что он тут используется для маппера формата времени (пригодится на рабочих с проектами с большими файлами конфигурациями)
```suggestion
    public Jackson2ObjectMapperBuilderCustomizer dateTimeMapper() {
```